CharlieBackend.Api.UnitTes/ValidatorsTests/UpdateScheduledEventDTOValidatorTests.cs

```diff
@@ -0,0 +1,163 @@
+﻿using CharlieBackend.Api.Validators.ScheduledEventDTOValidators;
+using CharlieBackend.Core.DTO.Schedule;
+using CharlieBackend.Data.Repositories.Impl.Interfaces;
+using FluentAssertions;
+using FluentValidation.Results;
+using Moq;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+namespace CharlieBackend.Api.UnitTest.ValidatorsTests
+{
+    public class UpdateScheduledEventDTOValidatorTests : TestBase
+    {
+        private UpdateScheduledEventDTOValidator _validator;
+        private readonly long validStudentGroupId = 1;
+        private readonly long validMentorId = 1;
+        private readonly long validThemeId = 1;
+        private readonly long notValidStudentGroupId = 0;
+        private readonly long notValidMentorId = 0;
+        private readonly long notValidThemeId = 0;
+        private readonly DateTime validStartDate = new DateTime(2020, 01, 01);
+        private readonly DateTime validEndDate = new DateTime(2020, 01, 01);
+        private readonly DateTime notValidStartDate = new DateTime(2020, 01, 01);
+        private readonly DateTime notValidEndDate = new DateTime(2019, 01, 01);
+        
+        public UpdateScheduledEventDTOValidatorTests()
+        {
+            _validator = new UpdateScheduledEventDTOValidator();
+        }
+
+        public UpdateScheduledEventDto Get_UpdateScheduledEventDTO(
+            long? mentorId = null,
+            long? studentGroupId = null,
+            long? themeId = null,
+            DateTime? eventStart = null,
+            DateTime? eventEnd = null)
+        {
+            return new UpdateScheduledEventDto
+            {
+                MentorId = mentorId,
+                StudentGroupId = studentGroupId,
+                ThemeId = themeId,
+                EventStart = eventStart,
+                EventEnd = eventEnd
+            };
+        }
+
+        [Fact]
+        public async Task UpdateScheduledEventDTOAsync_ValidData()
```

Comment:
We use this convention for unit tests naming MethodUnderTheTest_State_ShouldResult. Please find our examples in the solution.

Reply:
Also, try write test using a fluent validation test helper
![image](https://user-images.githubusercontent.com/15215380/112447179-ae407e00-8d59-11eb-8fa9-967d6cd432a8.png)

Reply:
Made naming according to the rules, added test helper add arrange/act/assert blocks
![Screenshot_9](https://user-images.githubusercontent.com/56502713/112451802-90294c80-8d5e-11eb-99dc-9a8aedacba9d.png)